integration_tests/fixtures/forseti/main.tf

```diff
@@ -135,15 +135,17 @@ resource "google_kms_crypto_key" "test-crypto-key" {
   rotation_period = "100000s"
 }
 
+
 provider "gsuite" {
   version     = "~> 0.1"
   credentials = "/workspace/release-silver.json"
   impersonated_user_email = "admin@silver.forsetisecurity.dev"
 }
 
 resource "gsuite_group" "devteam" {
-  email       = "automated-group-creation@silver.forsetisecurity.dev"
-  name        = "automated-group-creation@silver.forsetisecurity.dev"
+  email       = "gsuite-${random_pet.random_name_generator.id}@silver.forsetisecurity.dev"
```

Comment:
Can you please explain how this random name would fix the build?  What was the build failure?

Reply:
Build failed as the group name and email were not randomized. Names need to be randomized as after deleting the resource, new resource cannot be created with the same name for certain period of time.

Reply:
Got it.  Thanks for explaining.